packages/camera/camera/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.9.4+23
```

Comment:
Since we renamed the error code which is a breaking change, i think we should bump the major version. And since we are still in beta, major version is shifted to minor version, so it should be `0.10.0` here.

Reply:
Ah gotcha, thanks!

Reply:
Are the iOS errors already using these names? It would be good to not make two separate breaking changes to `camera`, on for Android and one for iOS.

Reply:
The iOS didn't handle these error before, so it's not a breaking change. Android and web will be breaking changes.

Reply:
In that case it would be much better to hold this PR for the moment, make the `camera_web` breaking change, and then have this PR include bumping the major version of `camera_web`, so that this PR is a single breaking change that brings all platforms into alignment.

Breaking changes in app-facing plugin packages are disruptive to clients, so should be minimized when possible.

Reply:
The web change is not ready yet. How about a minor bump for Android to land this PR first, and then have the web change to do the major bump?

Reply:
You can't land a breaking change as a minor version bump; we follow semver.

Reply:
@stuartmorgan Makes sense to me on holding this off. For now, though, made the changes if you can take a look! I wasn't sure how to reference Java code from Dart and vice versa, so if you have any feedback on that, I'd greatly appreciate it!

Reply:
@hellohuanlin I renamed the error code, and since it wasn't in your design, if you can take a look, that'd be great!